README.org

```diff
@@ -5,19 +5,24 @@
 
 * Orgzly Revived
 
-Orgzly is an outliner for taking notes and managing to-do lists.
+Orgzly Revived is a community-maintained version of Orgzly, as the development of the original app is no longer active.
 
-You can keep notebooks stored in plain-text and have them synchronized
-with a directory on your mobile device, SD card, WebDAV server or Dropbox.
+Orgzly Revived project aims to continue development of the Orgzly application, adding useful features, 
+fixing bugs, keeping up-to-date with technological evolution, and generally maintaining the 
+[Org Mode](https://orgmode.org/) support on Android.
+
+[Orgzly](https://orgzly.com/) is an outliner for taking notes and managing to-do lists. The source code
+[repository](https://github.com/orgzly/orgzly-android) is available on GitHub but is no longer actively 
+maintained. We have used this code as the base for Orgzly Revived.
```

Comment:
I think we should add a note about getting involved and transferring old issues and PRs as you brought up in your discussion post! https://github.com/orgzly-revived/orgzly-android-revived/discussions/2

```suggestion
maintained. We have used this code as the base for Orgzly Revived. Please feel free to get involved in the project on GitHub by contributing issues, ideas, or features! We generally plan to leave existing open issues in the original repository and reference them here as we address them.
```

Reply:
We should, at some point, add a Contributing doc. For now, I've created a separate section.
#45